src/NuGetForUnity/Editor/Ui/NugetWindow.cs

```diff
@@ -987,34 +857,39 @@ private void DrawInstalledHeader()
         /// </summary>
         private void DrawUpdatesHeader()
         {
-            var headerStyle = GetHeaderStyle();
+            var headerStyle = Styles.HeaderStyle;
 
             EditorGUILayout.BeginVertical(headerStyle);
```

Comment:
I replaced `BeginHorizontal/Vertical()` with `using(...)` for the most part. Some parts are left as they are because I thought that `BeginHorizontal/Vertical()` would be better in that part.